### ISSUE TITLE
Add progress logs to task wait command

### ISSUE THREAD

**Issue by talenor** [OctopusDeploy/cli#363]
In old CLI we have flag --progress that allow to see task logs in CI logs and they could be quite useful for troubleshooting.
Could the similar flag be added to the new CLI?

I've found a similar closed issue https://github.com/OctopusDeploy/cli/issues/227 but it was just closed without any explanation/suggested workaround

**Comment by QuentinScDS** [comment]
Yes, it's essential 😔

**Comment by vaiddas** [comment]
is there any plan to add this flag?

**Comment by mark-studer-compex** [comment]
Agreed, this is a very important item depending on the teams workflow.  We use Bitbucket pipelines to manage the deployment and developers don't have direct access to Octopus as all the configs are stored in the repo.  In testing out the new CLI this is a big loss which is keeping us from moving off the Octo CLI.

**Comment by Clare-Octopus** [comment]
Another customer reported this is essential for them and are not able to upgrade to the V6 ADO step versions until they can view the task logs in the build pipeline for V6 tasks.

Internal ticket - https://octopus.zendesk.com/agent/tickets/224353

**Comment by mark-studer-compex** [comment]
The new changes look very nice, once the workertools docker image (https://hub.docker.com/r/octopuslabs/workertools) is updated I'll try it out.

**Comment by vaiddas** [comment]
Hi @HuyPhanNguyen - Thanks for fixing this and now we have started using this flag. 

Quick question: Is there a way to view the progress in verbose mode? I can see Info, Warning, and Fatal errors.  

I can't figure out how to set verbose mode like we do in the Octopus Console. Am I missing something or it's not possible for now? 

Thanks,
